app/assets/stylesheets/main.scss

```diff
@@ -66,7 +66,25 @@ textarea#auction_description {
 }
 
 .github-issue {
-  white-space: pre-line;
+  h1 {
+	  line-height: 1.1em;
+	  margin-top: 0.2em;
+	  margin-bottom: 0.5em;
+  }
+
+  h2, h3, h4, h5, h6 {
+	  line-height: 1.1em;
+	  margin-top: 0.2em;
+	  margin-bottom: 0.1em;
+  }
+
+  p, li {
+	  line-height: 1.3em;
+  }
+
+  ul {
+	  margin-bottom: 0.5em;
+  }
 }
```

Comment:
The requirement was that it was "significantly reduced." The 30px was stated as "e.g." which means for example, not the requirement. The requirement was to remove the erroneous vertical spacing which was created by `white-space: pre-line;`. The Draft U.S. Web Design Standards defines the vertical white space and was created for optimal legibility.